test-two/src/index.js

```diff
@@ -3,24 +3,49 @@ import { getUsersForOrganisation } from "./services/github";
 
 const baseUrl = "https://api.github.com/";
 const organisationId = "google";
-const gitHubApiAuthToken = ""; // add your GitHub API OAuth key here to increase request limit
+const gitHubApiAuthToken = "cf33f5689e7e8399aadb5cbaae8ef51bc2be7c66"; // add your GitHub API OAuth key here to increase request limit
```

Comment:
Should this be committed to source control?

Reply:
No as its my OAuth Key, could be used maliciously